app/models/medicaid_application_attributes.rb

```diff
@@ -6,10 +6,57 @@ def initialize(medicaid_application:)
   def to_h
     {
       signature: medicaid_application.signature,
-    }.symbolize_keys
+      residential_address_county:
+        medicaid_application.residential_address.county,
+      residential_address_state: medicaid_application.residential_address.state,
+      residential_address_street_address: residential_or_homeless,
+      residential_address_zip: medicaid_application.residential_address.zip,
+      residential_address_city: medicaid_application.residential_address.city,
+      email: medicaid_application.email,
+      mailing_address_city: medicaid_application.mailing_address.city,
+      mailing_address_county: medicaid_application.mailing_address.county,
+      mailing_address_state: medicaid_application.mailing_address.state,
+      mailing_address_street_address:
+        medicaid_application.mailing_address.street_address,
+      mailing_address_zip: medicaid_application.mailing_address.zip,
+      receive_info_by_email:
+        bool_to_checkbox(medicaid_application.email.present?),
+      not_receive_info_by_email:
+        bool_to_checkbox(medicaid_application.email.blank?),
+      preferred_language: nil,
+    }.merge(phone_attributes).symbolize_keys
   end
 
   private
 
   attr_reader :medicaid_application
+
+  def bool_to_checkbox(statement)
+    if statement == true
+      "Yes"
+    end
+  end
+
+  def residential_or_homeless
+    if medicaid_application.stable_housing?
+      medicaid_application.residential_address.street_address
+    else
+      "Homeless"
+    end
+  end
+
+  def phone_attributes
```

Comment:
does it make sense to move this/bool_to_checkbox/other shared methods into a shared helper across attribute classes?

Reply:
done for at least Medicaid/Snap at 8c38fa05